scripts.py

```diff
@@ -0,0 +1,11 @@
+# This is a temporary workaround till Poetry supports scripts, see
+# https://github.com/sdispater/poetry/issues/241.
```

Comment:
I wish this was done soon enough with poetry, but they have been prioritizing with other things.